src/include/com/amazonaws/kinesis/video/webrtcclient/Stats.h

```diff
@@ -159,23 +158,19 @@ typedef struct {
                                      //!< The average interval between two consecutive connectivity checks sent can be calculated:
                                      //! (lastRequestTimestamp - firstRequestTimestamp) / requestsSent.
     UINT64 lastResponseTimestamp;    //!< Represents the timestamp at which the last STUN response was received on this particular candidate pair.
-    DOUBLE totalRoundTripTime;       //!< The sum of all round trip time (seconds) since the beginning of the session, based
+    UINT64 totalRoundTripTime;       //!< The sum of all round trip time (seconds) since the beginning of the session, based
                                      //!< on STUN connectivity check responses (responsesReceived), including those that reply to requests
                                      //!< that are sent in order to verify consent. The average round trip time can be computed from
                                      //!< totalRoundTripTime by dividing it by responsesReceived.
-    DOUBLE currentRoundTripTime;     //!< Latest round trip time (seconds)
-    DOUBLE availableOutgoingBitrate; //!< Total available bit rate for all the outgoing RTP streams on this candidate pair. Calculated by underlying
-                                     //!< congestion control
-    DOUBLE availableIncomingBitrate; //!< Total available bit rate for all the outgoing RTP streams on this candidate pair. Calculated by underlying
-                                     //!< congestion control
+    UINT64 currentRoundTripTime;     //!< Latest round trip time (seconds)
```

Comment:
round trip time is in seconds and it's an uint64. wouldnt it be 0 most of the time?

Reply:
Earlier, I had set it to DOUBLE to align with spec. Hence the confusion. I have changed it back to DOUBLE and value computation to seconds.